source/future.tex

```diff
@@ -1,4 +1,4 @@
-%!TEX root = std.tex
+nnn%!TEX root = std.tex
```

Comment:
What's `nnn`?

Reply:
A keyboard gone rogue.  Force-pushed the fix.